EXTRA.md

```diff
@@ -0,0 +1,30 @@
+Extra Information and Tips
+======
+
+## SSL Verification
+To disable SSL verification (for example, if a server links to another)
+Add to the server_settings table in bot.db:
+
+| server_id | setting | value |
+| --- | --- | --- |
+| 1 | ssl-verify  | false |
+
+---
+
+## SASL Authentication
```

Comment:
Worth also mentioning {"mechanism": "EXTERNAL"} which is (almost always) used to do client certificate authentication

Reply:
I'm completely unsure of how to do external SASL with the bot, might be an idea to throw it in there if you have time?

Reply:
it's literally the same as the `PLAIN` example but with `{"mechanism": "EXTERNAL"}`